basic.html

```diff
@@ -16,11 +18,15 @@
 <!-- Force latest IE rendering engine or ChromeFrame if installed -->
 <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"><![endif]-->
 <meta charset="utf-8">
-<title>jQuery File Upload Demo - Basic version</title>
+<title>Yearbook Upload</title>
```

Comment:
@uagarwal744 @punit121

Please change this to `Yearbook 2016 Photo Uploader | Students' Alumni Cell`